thesis-de/content-de.tex

```diff
@@ -1730,88 +1746,101 @@ \subsection{Definition eines Graphen}
 
-Auf der Weboberfläche werden dem User oder dem Administrator einzelne Graphen
-von z.B. CPU-Auslastung, RAM-Auslastung oder SSD-Schreibzugriffe angezeigt.
-Diese Graphen werden entweder in Linerar- oder Balkenform dargestellt. Lineare
-Graphen haben eine X- und Y-Achse und besitzen zusätzlich einen Anfang und
-einen wachsenden Endpunkt. Er wird mittels
-Metrik~\ref{section:Begriffserklärung}, welche die Weboberfläche aus den
-Datenhaltungssystemen abgreift, erstellt und kann vom Administrator ebenfalls
-nur für eine definierte Zeitspanne angezeigt werden. Balkendiagramme haben nur
-eine X-Achse, wo der gewünschte Analysewert eingetragen ist und werden z.B. für
-ausgefallene SSDs in einem bestimmten Zeitraum verwendet. Diese werden über die
-Metrik aus dem Datenhaltungssystem abgegriffen und dem Administrator über eine
-Übersicht angezeigt. Ein Graph- bzw. Balkendiagramm visualisiert immer eine
-oder mehrere Timeseries~\ref{section:Begriffserklärung}.
+Innerhalb der Datenvisualisierungslösung werden dem User oder dem Administrator
+verschiedene Graphen von beispielsweise CPU-Auslastung, RAM-Auslastung oder
+SSD-Schreibzugriffe visualisiert. Diese Graphen können entweder in Linerar-
+oder Balkenform dargestellt werden. Lineare Graphen haben eine X- und Y-Achse
+und besitzen zusätzlich einen Anfang und einen wachsenden Endpunkt. Er wird
+mittels Metrik (siehe Abschnitt~\ref{section:Begriffserklärung}), welche die
+Weboberfläche aus den Datenhaltungssystemen abgreift, erstellt und kann vom
+Administrator ebenfalls nur für eine definierte Zeitspanne angezeigt werden.
+Balkendiagramme haben nur eine X-Achse, wo der gewünschte Analysewert
+eingetragen ist und werden z.B. für ausgefallene SSDs in einem bestimmten
+Zeitraum verwendet. Diese werden über die Metrik aus dem Datenhaltungssystem
+abgegriffen und dem Administrator über eine Übersicht angezeigt. Ein Graph-
+bzw. Balkendiagramm visualisiert immer eine oder mehrere Timeseries (siehe
+Abschnitt~\ref{section:Begriffserklärung}).
 
 \subsection{Vorbereitung der Evaluierung}
 \label{subsec:vorbereiten_der_evaluierung_datenvisualisierung}
 Damit die Evaluierungen mittels einer Nutzwertanalyse der einzelnen
 Softwarelösungen durchgeführt werden können, müssen zuvor im ersten Schritt
 Kriterien mit dem Projektteam in einen Kriterienkatalog aufgenommen werden.
-Anschließend werden diesen Kriterien Gewichtungen zugewiesen. Bei den Kriterien
-musste darauf geachtet werden, dass diese während der Beurteilung der
-Anforderungen eine Realisierbarkeit besitzen. Nachdem die einzelnen Kriterien
-zusammen mit dem Projektteam erstellt worden sind, wurden diese im Anschluss
-von jedem einzelnen Projektmitglied auf die Mach- und Realisierbarkeit geprüft.
+Anschließend werden diesen Kriterien entsprechende Gewichtungen zugewiesen. Bei
+den Kriterien musste darauf geachtet werden, dass diese während der Beurteilung
+der Anforderungen eine Realisierbarkeit besitzen. Nachdem die einzelnen
```

Comment:
Realisierbarkeit stört mich ein wenig.

Kannst du nicht in 1 2 sätzen schreiben, was in diesem Projekt denn realisierbar wäre. 
muss während der Beurteilung beachtet werden, dass diese innerhalb dem Projekt zur Verfügung stehenden Zeit realisierbar sind. Dabei wird das prüfen, testen, realisieren etc. bla bla mit beachtet